src/Field.php

```diff
@@ -497,6 +500,42 @@ public function setNull(): self
         return $this;
     }
 
+    private function getDatabasePlatform(): Platforms\AbstractPlatform
+    {
+        return $this->getOwner()->persistence !== null
+            ? $this->getOwner()->persistence->getDatabasePlatform()
+            : new Persistence\NosqlPlatform();
+    }
+
+    /**
+     * @param mixed|void $value
+     */
+    public function getUnmanagedValue($value = null): ?string
```

Comment:
@georgehristov or `getDatabaseValue` to be consistent with DBAL naming?

unmanaged is used more in C, but "database" imply SQL persistence to me...

Reply:
For me the naming in order of preference
1. getPersistenceValue
2. getDatabaseValue
...

Reply:
👍 for `getPersistenceValue`

Reply:
I think I have choosen `unmanaged`/`maanged` because we need two names, eg. for both directions.

If we go with `getPersistenceValue`, `getPHPValue` as named by Doctrine https://github.com/doctrine/dbal/blob/3.1.2/src/Types/Type.php#L87 will not be ideal as persistence can be implemented in PHP as well.

Reply:
I dropped these new methods completely, just use persistence with the original names.